tasks/main.yml

```diff
@@ -1 +1,71 @@
-# role tasks
+- name: Gather system locale facts for idempotence checks
+  command: >
+    busctl --system get-property org.freedesktop.locale1
+    /org/freedesktop/locale1 org.freedesktop.locale1
+    {{ localectl_properties | join(' ') }}
+  register: localectl_busctl_gather
+  changed_when: false
+  become: "{{ localectl_become }}"
```

Comment:
Does this require elevated privileges to run?  On my system I can grab info with `busctl` using a non-privileged account.

Reply:
I had mixed experiences with getting information from the system dbus stuff. The short answer is "I'm not sure, so I'm playing it safe."

Reply:
That said, I'll remove it, since this is why we have CI, and my personal experience is effectively worthless as long as it works in CI.